docs/contributor/02-01-installation.md

```diff
@@ -1,4 +1,112 @@
+  - [Install on Kyma runtime](#install-on-kyma-runtime)
+    - [Lifecycle management of Keda Manager in Kyma](#lifecycle-management-of-keda-manager-in-kyma)
+  - [Run Keda module locally with Kyma and Lifecycle Manager on k3d](#run-keda-module-locally-with-kyma-and-lifecycle-manager-on-k3d)
+    - [Run Keda module locally on bare k3d](#run-keda-module-locally-on-bare-k3d)
+    - [Install Keda module on remote Kyma runtime](#install-keda-module-on-remote-kyma-runtime)
+
+
+Learn how to install Keda Manager locally (on k3d) or on your remote cluster.
+
+## Prerequisites
+
+- Access to a Kubernetes (v1.24 or higher) cluster or [k3d](https://k3d.io/v5.4.6/)
+- [Go](https://go.dev/)
+- [Docker](https://www.docker.com/)
+- [kubectl](https://kubernetes.io/docs/tasks/tools/)
+- [kubebuilder](https://book.kubebuilder.io/)
+
+## Install Keda Manager without Kyma 
+
+You can build and run the Keda Manager in the Kubernetes cluster without Kyma.
+For the day-to-day development on your machine, you don't always need to have it controlled by Kyma's Lifecycle Manager.
+
+Run the following commands to deploy Keda Manager on a target Kubernetes cluster (for example, on k3d):
+
+1. Clone the project.
+
+   ```bash
+   git clone https://github.com/kyma-project/keda-manager.git && cd keda-manager/
+   ```
+
+2. Set the Keda Manager image name.
+
+   > NOTE: You can use local k3d registry or your Docker Hub account to push intermediate images.  
+   ```bash
+   export IMG=<DOCKER_USERNAME>/custom-keda-manager:0.0.2
+   ```
+
+3. Verify the compatibility.
+
+   ```bash
+   make test
+   ```
+4. Build and push the image to the registry.
+
+   ```bash
+   make module-image
+   ```
+5. Create a target Namespace.
+
+   ```bash
+   kubectl create ns kyma-system
+   ```
+
+6. Deploy Keda Manager.
+
+   ```bash
+   make deploy
+   ```
+
+7. Verify if Keda Manager is deployed.
+
+   ```bash
+   kubectl get deployments -n kyma-system
+   ```
+
+   You should get a result similar to this example:
+
+   ```
+   NAME                             READY   UP-TO-DATE   AVAILABLE   AGE
+   keda-manager            1/1     1            1           1m
+   ```
+## Build and publish images manually
+
+1. Export the required environmental variables
+
+   ```
+   export IMG="IMG"           // Keda Manager's image
+   export REGISTRY="REGISTRY" // the OCI registry the module will be published to
+   ```
+
+2. Run the following recipe to build and publish module
+
+   ```
+   make module-build \
+     IMG=${IMG} \
+     REGISTRY={REGISTRY}
+   ```
+
+## Install Keda module manually
+
+1. Apply the following script to install Keda Manager:
+
+   ```bash
+   kubectl create ns kyma-system
+   kubectl apply -f https://github.com/kyma-project/keda-manager/releases/latest/download/keda-manager.yaml
+   ```
+
+2. To get KEDA installed, apply the sample Keda CR:
+
+   ```bash
+   kubectl apply -f config/samples/operator_v1alpha1_keda_k3d.yaml
+   ```
 
 ## Install on Kyma runtime
```

Comment:
I believe that this section is neiither relevant for OS users nor for SKR users. Morover, for SKR users it may confuse them, as they will use the internal documentation for how to enable modules on kyma runtimes with lifecycle manager.

Lets delete this section ("Install on Kyma runtime")